runtime-complexity/runtime.js

```diff
@@ -25,7 +25,7 @@ const evaluate = function(f, _ns) {
     })
   }
 
-  var points = []
+  const points = []
```

Comment:
This should be `let` instead of `const`.

Reply:
But same thing has const in other parts of the file.

Reply:
@Kundan28  Please insert the lines to which you are pointing. 
If we make `points` a `const` variable then it can't be mutated in the lines 37-40
`points.push({
      x: n,
      y: timeMs
    });`

Reply:
Okay.

Reply:
@Aayush-1999
But const just means that the variable cannot be reassigned, not that we cannot manipulate it.
Pushing into the array will mutate it whether or not it is declared as let or const. i declared it as const so that points array count not be reassigned.

Reply:
Arrays and objects can be manipulated and still be const. it would only be let if you assigned a brand new array or object, or a diff data type,  to the variable.